lib/tasks/create_fake_data.rake

```diff
@@ -0,0 +1,89 @@
+namespace :db do
+  namespace :seed do
+    desc 'Generate fake patient entries for data wranglers'
+    task :create_fake_data => :environment do
+
+        users = User.all
+        clinics = Clinic.all
+
+        5000.times do |idx|
+          flag = (rand(10) % 5 == 0)
```

Comment:
A thing that we appreciate about the seeds file is that it spits out consistent data every time -- when we're faking it, that's kind of an asset, since you know the base layer is consistent when you're just screwing around. You've already got the modulo set up here, so what do you think about doing `idx % 5 == 0`?

Reply:
Love it! Also, gosh, so much of this beautiful logic was laid out in your starter code, so I have you to thank for all this Ruby brilliance! SO MUCH LEARNIN HAPPENED

Reply:
Love it! Thank you.